### DISCUSSION TITLE
Huawei 8X Max Brightness Issue 50% when set to 100%

### DISCUSSION THREAD

**Discussion by nation888** [phhusson/treble_experimentations#2447] [Q&A]
Model: ARE-AL00 Horror 

Trying for years to unlock the boot loader. Finally unlocked the bootloader by HCU. Tough find, but found the TWRP recovery. Now the GSI challenge. I installed the 12L and everything seemed to work except the brightness/backlight. The brightness is set to 100% and it is at 50%. Tried using apps like Lux, Phh settings [backlight] with no success.  Could not find the BuildProp value/line to edit. I'm at a loss with this mystery. 

After trying almost all the GSI. The [AOSP 10.0 v222] and other GSI Android 10 variants the brightness works perfectly. All the other GSIs afterwards found there are backlight light issues. What is different in this Android 10 version and the others? Is there or was there a patch?  (I should mention I tested the Pixel Experience 10 and strangely had the same is brightness issue).  

Thanks in advance

**Comment by zirangua** [comment]
If you enable Phh Treble Settings - Misc features, selecte force alternative backlight scale... with this option enable the backlight works...
Well this was probed on red me note 9
I hope this work for you
